safe-logging/build.gradle

```diff
@@ -1 +1,5 @@
 apply from: "${rootDir}/gradle/publish-jar.gradle"
+
+dependencies {
+    api 'com.google.errorprone:error_prone_annotations'
```

Comment:
Are we okay with this?

Reply:
will it break if we make it an implementation dependency?

Reply:
That should work, it may break if we remove the api dep from safe-logging:preconditions, would you prefer if I retain the `api` dep type there?

Reply:
nvm, implementation dependency will not trigger error-prone to fail on dependent code